Bug: __nameid_ directory should not be parsed

As discussed in PR #22, see also nameid-fix branch:

In a MSG file the __nameid_... directory represents named properties, which are as of right now not supported by this library. It tries to parse the entries in this directory using the normal property parser though, which creates conflicts and is wrong. You can also refer to the Microsoft documentation for this: [MS-OXMSG].

I had several emails where an id in this directory collided with the property id for HTML-Content (10130102), which caused RTF-Emails to incorrctly report this invalid content as the HTML body, which prevented the RTF Conversion from being read.

![grafik](https://user-images.githubusercontent.com/13820214/72885510-5dbe3280-3d08-11ea-8d14-78b4f0eb9418.png)

The fix in #22 could not be merged because of Tests failing! I can not reproduce this though, because my tests always fail with this error:

```
[ERROR] Tests run: 16, Failures: 0, Errors: 16, Skipped: 0, Time elapsed: 0.07 s <<< FAILURE! - in org.simplejavamail.outlookmessageparser.HighoverEmailsTest
[ERROR] testUnicodeMessage(org.simplejavamail.outlookmessageparser.HighoverEmailsTest)  Time elapsed: 0.009 s  <<< ERROR!
java.lang.Error:
Unresolved compilation problems:
        OutlookMessageAssert cannot be resolved
        OutlookMessageAssert cannot be resolved
        OutlookMessageAssert cannot be resolved
        OutlookMessageAssert cannot be resolved
        The method normalizeText(String) is undefined for the type HighoverEmailsTest

        at org.simplejavamail.outlookmessageparser.HighoverEmailsTest.testUnicodeMessage(HighoverEmailsTest.java:520)
```

--- a/src/main/java/org/simplejavamail/outlookmessageparser/OutlookMessageParser.java
+++ b/src/main/java/org/simplejavamail/outlookmessageparser/OutlookMessageParser.java
@@ -176,6 +176,9 @@ public class OutlookMessageParser {
 				} else if (de.getName().startsWith("__recip_version1.0")) {
 					// a recipient entry has been found (which is also a directory entry itself)
 					checkRecipientDirectoryEntry(de, msg);
+				} else if (de.getName().startsWith("__nameid_version1.0")) {
+					// a named property mapping directory has been found. We have to ignore it because 
+					// we can not parse it yet and it creates conflicts with normal parsing see also [MS-OXMSG]
 				} else {
 					// a directory entry has been found. this node will be recursively checked
 					checkDirectoryEntry(de, msg);

--- a/src/test/java/org/simplejavamail/outlookmessageparser/HighoverEmailsTest.java
+++ b/src/test/java/org/simplejavamail/outlookmessageparser/HighoverEmailsTest.java
@@ -160,7 +160,7 @@ public class HighoverEmailsTest {
 				createRecipient("'rob@thegopedal.com'", "rob@thegopedal.com"));
 		OutlookMessageAssert.assertThat(msg).hasNoOutlookAttachments();
 		assertThat(msg.getBodyText()).isNotEmpty();
-		assertThat(msg.getBodyHTML()).isNotEmpty();
+		assertThat(msg.getBodyHTML()).isNull();
 		assertThat(msg.getBodyRTF()).isNotEmpty();
 		assertThat(normalizeText(msg.getBodyText())).isEqualTo("Another test email to obtain raw data for Jean-Philippe.\n"
 				+ "\n"
@@ -249,7 +249,7 @@ public class HighoverEmailsTest {
 				+ "\n");
 		/* /nested message */
 		assertThat(msg.getBodyText()).isNotEmpty();
-		assertThat(msg.getBodyHTML()).isNotEmpty();
+		assertThat(msg.getBodyHTML()).isNull();
 		assertThat(msg.getBodyRTF()).isNotEmpty();
 		assertThat(normalizeText(msg.getBodyText())).isEqualTo("Mail in mail.\n"
 				+ "\n"
@@ -480,7 +480,7 @@ public class HighoverEmailsTest {
 		List<OutlookAttachment> outlookAttachments = msg.getOutlookAttachments();
 		assertThat(outlookAttachments).isEmpty();
 		assertThat(msg.getBodyText()).isNotEmpty();
-		assertThat(msg.getBodyHTML()).isNotEmpty();
+		assertThat(msg.getBodyHTML()).isNull();
 		assertThat(msg.fetchCIDMap()).isEmpty();
 		assertThat(msg.fetchTrueAttachments()).isEmpty();
 		assertThat(normalizeText(msg.getBodyText())).isEqualTo(" \n" +
@@ -636,7 +636,7 @@ public class HighoverEmailsTest {
 		assertThat(msg.getConvertedBodyHTML()).contains("cid:image001.png");
 		assertThat(msg.getConvertedBodyHTML()).contains("cid:image002.png");
 		assertThat(msg.getBodyText()).isNotEmpty();
-		assertThat(msg.getBodyHTML()).isNotEmpty();
+		assertThat(msg.getBodyHTML()).isNull();
 		assertThat(normalizeText(msg.getBodyText())).isEqualTo(" \n"
 				+ " \n"
 				+ "Van: Microsoft Outlook\n"

